docs/project_setup_and_build.md

```diff
@@ -0,0 +1,230 @@
+# Project structure and build
+
+## Overview
+
+In this chapter, you will get to know the anatomy of the standard BigFlow project, artifacts, and CLI commands. If you are interested
+in the details of the interface itself, go to [the CLI chapter](./cli.md).
+
+The BigFlow build system packages your processing logic into a Docker image. Thanks to this approach, you can create any
```

Comment:
The BigFlow build system packages your processing logic into a Docker image
->
The BigFlow build command packages your processing code into a Docker image

Reply:
done